ggml/src/ggml-metal.m

```diff
@@ -727,7 +758,7 @@ static void ggml_metal_free(struct ggml_backend_metal_context * ctx) {
 // the assumption is that there is 1-to-1 mapping between the host and device memory buffers, so we can find the
 // Metal buffer based on the host memory pointer
 //
-static id<MTLBuffer> ggml_metal_get_buffer(struct ggml_tensor * t, size_t * offs) {
+static id<MTLBuffer> ggml_metal_get_buffer(const struct ggml_tensor * t, size_t * offs) {
```

Comment:
I am not convinced that this change is correct since it return a non-const buffer taken from a const tensor.